src/api/emails/send-application-submitted-emails/index.ts

```diff
@@ -13,44 +14,90 @@ import { SuccessResponse, ApplicationSubmissionEmailVariables, Application } fro
  */
 const send = async (application: Application, csvPath: string): Promise<SuccessResponse> => {
   try {
-    const { referenceNumber, owner, company, buyer, policyAndExport } = application as Application;
+    const { referenceNumber, owner, company, buyer, policyAndExport, business } = application as Application;
+
+    const { businessContactDetail } = business;
 
     // generate email variables
     const { email } = owner;
 
-    const sendEmailVars = {
-      emailAddress: email,
-      name: getFullNameString(owner),
+    // shared variables for sending email
+    const sharedEmailVars = {
       referenceNumber,
       buyerName: buyer.companyOrOrganisationName,
       buyerLocation: buyer.country?.name,
       companyName: company.companyName,
       requestedStartDate: formatDate(policyAndExport.requestedStartDate),
+    };
+
+    // email variables for sending email to application owner of application
+    const sendEmailVars = {
+      ...sharedEmailVars,
+      name: getFullNameString(owner),
+      emailAddress: email,
     } as ApplicationSubmissionEmailVariables;
 
-    // send "application submitted" email receipt to the applicant
-    const accountSubmittedResponse = await sendEmail.applicationSubmitted.account(sendEmailVars);
+    // email variables for sending email to business contact named on application
+    const sendContactEmailVars = {
+      ...sharedEmailVars,
+      name: getFullNameString(businessContactDetail),
+      emailAddress: businessContactDetail.email,
+    } as ApplicationSubmissionEmailVariables;
+
+    // checks if application owner email on application is the same as contact email provided
+    const isOwnerSameAsContact = isOwnerSameAsBusinessContact(email, businessContactDetail.email);
+
+    console.info('Sending application submitted email to application account owner: ', sendEmailVars.emailAddress);
+    // send "application submitted" email receipt to the applcation owner applicant
+    const accountSubmittedResponse = await sendEmail.application.submittedEmail(sendEmailVars);
 
     if (!accountSubmittedResponse.success) {
       throw new Error('Sending application submitted email to owner/account');
     }
 
+    /**
+     * if the contact email address is different to the application owner applicant
+     * then it sends the same "application submitted" email receipt to the contact email address
+     */
+    if (!isOwnerSameAsContact) {
```

Comment:
`isOwnerSameAsBusinessContact`

Reply:
cant have the variable the same name as the function